docs/core/whats-new/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: What's new in .NET Core 2.0
+description: Learn about the new features found in .NET Core.
+keywords: .NET, .NET Core
+author: rpetrusha
+ms.author: ronpet
+ms.date: 08/07/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+---
+# What's new in .NET Core
+
+.NET Core 2.0 includes enhancements and new features in the following areas:
+
+- [Tooling](#tooling)
+- [Language support](#language-support)
+- [Platform improvements](#platform-improvements)
+- [API changes](#api-changes)
+- [Visual Studio integration](#visual-studio-integration)
+- [Documentation improvements](#documentation-improvements) 
+
+## Tooling
+
+**`dotnet restore` runs implicitly**
+
+In previous versions of the .NET Core, you had to run the [dotnet restore](../tools/dotnet-restore.md) command to download dependencies immediately after you created a new project with the [dotnet new](../tools/dotnet-new.md) command, as well as whenever you added a new dependency to your project. In .NET Core 2.0, `dotnet restore` runs implicitly when the `dotnet new` command executes. It also runs implicitly if dependencies need to be updated when other commands, such as the `run`, `build`, and `publish` commands, execute.
+
+You can also disable the automatic invocation of `dotnet restore` by passing the `--no-restore` switch to the `new`, `run`, `build`, `publish`, `pack`, and `test` commands. 
+
+**Retargeting to .NET Core 2.0**
```

Comment:
By reading @scottaddie's [article](https://docs.microsoft.com/en-us/aspnet/core/migration/1x-to-2x/) for the ASP.NET docs, I feel like this section is out of place. We can leave it here for now, but I think we should have a migration topic in our side that would cover \<TargetFramework>, global.json, package references and tools and leave the rest on the asp.net docs. @Rick-Anderson how would we reconcile the two since we don't want to have a lot of duplicate content?